generated/nidcpower/nidcpower/session.py

```diff
@@ -4756,6 +4988,94 @@ def _parse_channel_count(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return int(number_of_channels_ctype.value)
 
+    @ivi_synchronized
+    def perform_lcr_open_compensation(self, additional_frequencies=None):
+        r'''perform_lcr_open_compensation
+
+        Generates open compensation data for LCR measurements based on a default set of test frequencies and, optionally, additional frequencies you can specify.
+
+        You must physically configure an open LCR circuit to use this method to generate valid open compensation data.
+
+        When you call this method:
+
+        -  The open compensation data is written to the onboard storage of the instrument. Onboard storage can contain only the most recent set of data.
+        -  Most NI-DCPower properties in the session are reset to their default values. Rewrite the values of any properties you want to maintain.
+
+        To apply the open compensation data you generate with this method to your LCR measurements, set the lcr_open_compensation_enabled property to **TRUE**.
+
+        Corrections for frequencies other than the default frequencies or any additional frequencies you specify are interpolated.
+
+        Note:
+        This method is not supported on all devices. For more information about supported devices, search ni.com for Supported Methods by Device.
+
+        Tip:
+        This method can be called on specific channels within your :py:class:`nidcpower.Session` instance.
+        Use Python index notation on the repeated capabilities container channels to specify a subset,
+        and then call this method on the result.
+
+        Example: :py:meth:`my_session.channels[ ... ].perform_lcr_open_compensation`
+
+        To call the method on all channels, you can call it directly on the :py:class:`nidcpower.Session`.
+
+        Example: :py:meth:`my_session.perform_lcr_open_compensation`
+
+        Args:
+            additional_frequencies (array.array("d")): Defines a further set of frequencies, in addition to the default frequencies, to perform the compensation for. You can specify <=200 additional frequencies.
```

Comment:
We typically let users pass array-like data as `list`, e.g.:
https://github.com/ni/nimi-python/blob/b1bdc6b0a5bed482df775a37968575c39ac05866/generated/nidcpower/nidcpower/session.py#L5453

This enables duck typing and allows users to pass any object that behaves like a list.

Reply:
I've changed the type of `additional_frequencies` to "list of float" in my latest commit